lib/rules/eol-last.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
                 const sourceCode = context.getSourceCode(),
                     src = sourceCode.getText(),
                     location = {
-                        column: 1,
+                        column: lodash.last(sourceCode.lines).length,
```

Comment:
Given that `context.report` is supposed to use zero-based columns, should this be `lodash.last(sourceCode.lines).length - 1`? (I might be mistaken here.)

Actually, maybe it should be `.length` when a newline is absent, since that's where we would want to insert a newline in auto-fix... Yeesh. This could be a can of worms.